packages/e2e/test/config.ts

```diff
@@ -0,0 +1,121 @@
+import * as envalid from 'envalid';
+import {
+  AssetProvider,
+  Cardano,
+  ChainHistoryProvider,
+  NetworkInfoProvider,
+  RewardsProvider,
+  StakePoolProvider,
+  TxSubmitProvider,
+  UtxoProvider
+} from '@cardano-sdk/core';
+import { FaucetProvider } from '../src/FaucetProvider';
+import { KeyManagement } from '@cardano-sdk/wallet';
+import { LogLevel, createLogger } from 'bunyan';
+import { Logger } from 'ts-log';
+import {
+  assetProviderFactory,
+  chainHistoryProviderFactory,
+  faucetProviderFactory,
+  keyManagementFactory,
+  networkInfoProviderFactory,
+  rewardsProviderFactory,
+  stakePoolProviderFactory,
+  txSubmitProviderFactory,
+  utxoProviderFactory
+} from '../src/factories';
+import memoize from 'lodash/memoize';
```

Comment:
Just a note that `waitFor` in e2e config is gone @rhyslbw . I'm fine with it if the tests are reliable as it was causing some problems anyway.

Reply:
Yes, I read it was giving issues. It was being used to sort of ping blockfrost, blocking until it respond and then logging a message, but I think if there is a connectivity issue with blockfrost we can let the tests catch it. Of course I can added it back, let me know.

Reply:
That package would be more useful when we're starting our local services at the same time as running the tests, but we could also just manage the startup sequence appropriately.